Source/WTF/wtf/FileSystem.cpp

```diff
@@ -834,7 +834,7 @@ String pathByAppendingComponents(StringView path, const Vector<StringView>& comp
 
 #endif
 
-#if !OS(WINDOWS) && !PLATFORM(COCOA)
+#if !OS(WINDOWS) && !PLATFORM(COCOA) && !PLATFORM(PLAYSTATION)
 
 String createTemporaryDirectory()
```

Comment:
Did you confirm the function is not used in PlayStation platform?

Reply:
createTemporaryDirectory() is not currently in use in PlayStation.